crates/setup/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+/// Sets up tracing.
+pub fn tracing() {
+    tracing_subscriber::fmt::init();
```

Comment:
This setup could become quite involved in the future, that's why I created an external crate to do these kinds of "shared" setup routines.

Eventually we may add more functions to this crate.